lib/lita/adapters/slack/rtm_connection.rb

```diff
@@ -31,8 +31,15 @@ def initialize(robot, config, team_data)
           @websocket_url = team_data.websocket_url
           @robot_id = team_data.self.id
 
-          @slack_users = team_data.users
```

Comment:
Doesn't this need to be set so that the values can be called through `attr_reader`?

Reply:
Good catch! I'll actually remove that `attr_reader`, since it's no longer used.